### ISSUE TITLE
Problem with installing Debian

### ISSUE THREAD

**Issue by vik24** [ChrisTitusTech/Debian-titus#96]
Hi,I'm a total beginner to Linux. I'm installing the system without a root user. Just as a user who will use 'sudo'. When I go to Software selection I uncheck all options and leave only 'standard system utlilties'. I follow these commands from github:git clone https://github.com/ChrisTitusTech/debian-tituscd debian-titussudo ./install.shUnfortunately after a long time, **
**I get this message displayed:"You need to be a member of the sudo group to run me**!"**Previously it displays me this:*Using aptUsage: grep [OPTION] ... PATTERNS [FILE] ...Try 'grep -- help' for more information.*
**Do you know how to fix it/cause it to install completely correctly and successfully though?**

[https://imgur.com/a/WdinoAE](url)

**Comment by crazycheetah42** [comment]
For some reason, it displays that for everyone but it still installs properly. Don't worry about that. Just run sudo reboot.

**Comment by p3nguin-kun** [comment]
```
usermod -aG sudo [username]
```

**Comment by crazycheetah42** [comment]
This can be closed, I think.

**Comment by bckelley** [comment]
@vik24 I've only gotten this on debian 12 when assigning a password to the root user, when not assigning a root password I don't get this and am able to run sudo commands just fine.

If you run the install script does it work?

**Comment by crazycheetah42** [comment]
Yeah, I think it happens when you set a password for the root user. But you shouldn't be doing that anyway because I've heard it's a security risk.
Besides, the error happens for no reason and the desktop fully installs if you reboot it.
